acceptance/acceptance_test.go

```diff
@@ -0,0 +1,275 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	_ "embed"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"path"
+	"path/filepath"
+	"regexp"
+	"testing"
+
+	"github.com/cucumber/godog"
+)
+
+type (
+	inputFileName  struct{}
+	configFileName struct{}
+	testStateKey   struct{}
```

Comment:
Use one context state? i.e. place input and config file paths in the `testState`?

Reply:
https://github.com/enterprise-contract/ec-policies/pull/866